SCClassLibrary/JITLib/ProxySpace/BusPlug.sc

```diff
@@ -256,6 +256,7 @@ BusPlug : AbstractFunction {
 			^this
 		};
 		if(bus.rate == \control) { "Can't monitor a control rate bus.".warn; monitor.stop; ^this };
+		group = group ?? {this.homeServer.defaultGroup};
```

Comment:
hm... good question. of course, function evaluation is costs as well.
Also, `TempoClock.default` is an instance variable of `TempoClock` so it only just returns the value, thus, in the case below, there's not much difference. compare:

```
bench{50000.do{[nil, 1].choose ? TempoClock.default}};
bench{50000.do{[nil, 1].choose ?? {TempoClock.default}}};
```

whereas in the upper case, `defaultGroup` is a method of server which requires computation, hence we only want to evaluate it when it is needed.